app/views/layouts/_show_links_to_children_of_page_with_title.html.erb

```diff
@@ -0,0 +1,13 @@
+<div>
+  <% if (featured_page = find_page_with_title(title_of_page_to_render)) %>
+
+    <h2 class="govuk-heading-l"><%= title_of_page_to_render %></h2>
+    <p class="govuk-body"><a class="govuk-link" href="<%= path_for_this_page(featured_page) %>">Overview</a></p>
+
+    <% (featured_page&.children || []).each do |child| %>
+        <p class="govuk-body"><a class="govuk-link" href="<%= path_for_this_page(child) %>"><%= child.title %></a></p>
+    <% end %>
+  <% else %>
+    <p class="govuk-error-summary"><%= "Error: Could not find page called #{title_of_page_to_render}" %></p>
```

Comment:
Where would this error appear on the page? Would it be better to not show an error if it appears half way down the page?

Reply:
I've added an error higher up now.

Reply:
![Screenshot 2021-03-08 at 15 00 06](https://user-images.githubusercontent.com/3352035/110341019-cf972f80-8021-11eb-99ca-b7378654ca91.png)
Here you go. Really this will only happen if the editors change the name of the 'Get Help Page'.